docs/android/v2/plugins/Getting-started-virtual-background.mdx

```diff
@@ -0,0 +1,154 @@
+---
+title: Getting Started - Virtual Background
+nav: 12
+---
+
+## Introduction
+
+Virtual background plugin helps in customising one’s background. The customising options is replacing the background with a static image.
+This guide provides an overview of usage of the virtual background plugin of 100ms.
+
+![VirtualBackground](/docs/v2/vb.gif)
+
+## Supported Android API Levels
+
+Virtual background is currently support all android os with API level 16 and higher. It is built for armeabi-v7a, arm64-v8a, x86, and x86_64 architectures.
+
+## Prerequisites
+
+Familiarity with Android Studio and the [fundamentals](https://developer.android.com/training/basics/firstapp) of android apps.
+
+## Basic Concepts
+
+-   `image background` - This is where the plugin will replace the video background with the image provided by user.
+
+### Add dependency
+
+-   Add the 100ms VirtualBackground and sdk dependency to your app-level `build.gradle` [![](https://jitpack.io/v/100mslive/android-sdk.svg)](https://jitpack.io/#100mslive/android-sdk).
+
+```json{3,4}:build.gradle
+dependencies {
+ // See the version in the jitpack badge above.
+ implementation 'com.github.100mslive:android-sdk-virtual-background:x.x.x'
+ implementation 'com.github.100mslive:android-sdk:x.x.x'
+    }
+```
+### Instantiate HMSVirtualBackground
+
+Instantiate the HMSVirtualBackground like this:
```

Comment:
See /android/v2/features/Join for how android sdk apis are explained. Update this to similar form